test/CMakeLists.txt

```diff
@@ -406,3 +406,11 @@ add_subdirectory(ci)
 if(WIN32)
   add_subdirectory(performance)
 endif()
+
```

Comment:
Please add a short comment describing what this test target is for, and make a note that this requires an install tree (or else `tiledb_version.h` will not exist)